src/gluonts/model/npts/_predictor.py

```diff
@@ -169,11 +168,11 @@ def __init__(
         use_default_time_features: bool = True,
         num_default_time_features: int = 1,
         feature_scale: float = 1000.0,
-        num_parallel_samples: int = 100,
+        num_samples: int = 100,
```

Comment:
My proposal was to remove this from the constructor of predictors, and keep it only as an argument to the `predict` method, see #301